src/registrar/assets/src/sass/_theme/_header.scss

```diff
@@ -426,3 +431,31 @@ $-expand-less-icon: (
         visibility: hidden;
     }
 }
+
+#organizations-submenu {
```

Comment:
We talked about this, but I wonder if this specific styling could go in a file in a less global location than themes (or that this could be a more global styling for future submenus instead of attached to an id)